library-model-loader/src/main/java/edu/ucr/cs/riple/librarymodel/LibraryModelLoader.java

```diff
@@ -120,4 +130,29 @@ public ImmutableSet<MethodRef> nonNullReturns() {
   public ImmutableSetMultimap<MethodRef, Integer> castToNonNullMethods() {
     return ImmutableSetMultimap.of();
   }
+
+  @Override
+  public ImmutableSet<FieldRef> nullableFields() {
+    return nullableFields;
+  }
+
+  @Override
+  public ImmutableList<StreamTypeRecord> customStreamNullabilitySpecs() {
+    return LibraryModels.super.customStreamNullabilitySpecs();
+  }
```

Comment:
Isn't this override unnecessary?  It just calls the super method?

Reply:
Thanks for catching this, I really don't know what I was thinking 🤦🏻‍♂️ de61967d01ff7e45865ce4f03a9cd42c63e3a2f2